server/sonar-server/src/main/java/org/sonar/server/es/EsClientProvider.java

```diff
@@ -56,13 +56,14 @@ public EsClient provide(Settings settings) {
         esSettings.put("client.transport.sniff", true);
         nativeClient = TransportClient.builder().settings(esSettings).build();
         Arrays.stream(settings.getStringArray(ProcessProperties.CLUSTER_SEARCH_HOSTS))
-          .map(Host::parse)
-          .forEach(h -> h.addTo(nativeClient));
+          .map(HostAndPort::fromString)
```

Comment:
@sns-seb I'll add ITs for this. The main issue was that EsClientProvider *in cluster mode* required to have `sonar.cluster.search.hosts=192.168.0.10:9001,192.168.0.11:9001` and `sonar.cluster.search.hosts=192.168.0.10,192.168.0.11` was not allowed making configuration of cluster more harder.
The main modification is here with HostAndPort::fromString function that will automatically add the default port.

When SonarQube is installed standalone there is no need for such change.